src/site/content/en/blog/get-installed-related-apps/index.md

```diff
@@ -228,8 +232,8 @@ relatedApps.forEach((app) => {
 {% Aside %}
 `getInstalledRelatedApps()` is currently supported on the following platforms:
 
-* Android: Chrome 80+
-* Windows: Edge (coming soon), Chrome (coming soon)
+* Android: Chrome 80 and later
+* Windows: Edge Chrome 85 and later; Chrome (coming soon)
```

Comment:
@petele & @jpmedley another instance of mixing Edge and Chrome here (Edge Chrome 85)